partcad/src/partcad/ai_ollama.py

```
@@ -33,22 +32,22 @@
 models_pulled = {}
 
 
-def ollama_once():
+def ollama_once(num_thread):
```

Ruff (0.8.2)

Missing return type annotation for public function `ollama_once`

Add return type annotation: `bool`

(ANN201)